### PR TITLE
Add qotp extension

### PR DESCRIPTION
**Pull request by ariesly15**
## Screencast

Metadata

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

* You can use [getProgressIcon](https://developers.raycast.com/utilities/icons/getprogressicon) instead of the images

* At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.  

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Review by pernielsentikaer — COMMENTED** [review]
Could you please remove all the icons not needed anymore 😊

File: extensions/qotp/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# QOTP
+
+## pin 获取
+
+qotp.corp 平台,用户输入的六位密码
+
+## [secret 获取教程](https://sourl.cn/4NfR9x)
```
Comment (pernielsentikaer) [inline extensions/qotp/README.md:7]: At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.

File: extensions/qotp/package.json [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    }
+  ],
+  "preferences": [
+    {
+      "required": true,
+      "name": "pin",
+      "title": "PinCode",
+      "type": "password",
+      "description": "qotp.corp 平台中用户设置的6位密码",
+      "placeholder": "qotp.corp 平台中用户设置的6位密码"
```
Comment (pernielsentikaer) [inline extensions/qotp/package.json:29]: At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.

File: extensions/qotp/package.json [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+      "type": "password",
+      "description": "Your Secret",
+      "placeholder": "Enter Your Secret"
+    },
+    {
+      "required": false,
+      "name": "passphrase",
+      "title": "passphrase(跳板机登录线上机器的密码)",
+      "type": "password",
+      "description": "跳板机登录线上机器的密码"
```
Comment (pernielsentikaer) [inline extensions/qotp/package.json:44]: At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.

File: extensions/qotp/package.json [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+    },
+    {
+      "required": false,
+      "name": "genCodeCount",
+      "title": "生成的code数量",
+      "type": "textfield",
+      "default": "3",
+      "description": "生成的code数量",
+      "placeholder": "请填写数字"
```
Comment (pernielsentikaer) [inline extensions/qotp/package.json:53]: At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.

File: extensions/qotp/plugins.json [hunk 5]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "QOTP",
+  "description": "生成 OTP 六位动态码",
```
Comment (pernielsentikaer) [inline extensions/qotp/plugins.json:3]: At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.

File: extensions/qotp/src/menu-bar-qotp.tsx [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+//   "name": "menu-bar-qotp",
+//   "title": "QOTP",
+//   "description": "QOTP is a simple tool to generate OTP code",
+//   "mode": "menu-bar",
+//   "interval": "1s"
+// }
+
+export default function Command() {
+  return (
+    <MenuBarExtra icon="otp.png" tooltip="点击查看OTP">
```
Comment (pernielsentikaer) [inline extensions/qotp/src/menu-bar-qotp.tsx:15]: At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

* The logo is not a good extension icon due to the transparency, you can maybe use [icon.ray.so](https://icon.ray.so/) to generate one 😊 

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by pernielsentikaer** [comment]
Should I help you with a proper logo?

**Comment by ariesly15** [comment]
> Should I help you with a proper logo?

Yes, I do. Thank you👏

**Comment by pernielsentikaer** [comment]
What do you think about this one?

qotp

**Comment by ariesly15** [comment]
> What do you think about this one?
> 
> qotp

I think it's great.

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**